### ISSUE TITLE
Why the AJAX response is not returned when using PHP threads

### ISSUE THREAD

**Issue by hAbd0u** [byjg/php-phpthread#4]
I am finding it very strange that after the `startPool()` method no result will get returned to the AJAX handler, can you explain to me this why?
Here is a [reproduce](https://github.com/hAbd0u/AjaxPhpMultithreadTest) to the issue I am facing now.

**Comment by byjg** [comment]
The algorithm is working as programmed. Each thread creates a file and inside the file is the result of the thread execution. 

<img width="824" alt="image" src="https://user-images.githubusercontent.com/981924/175753048-b8007c7f-8cbe-45b7-8012-a89334cf00ab.png">

I can't understand the issue.

**Comment by hAbd0u** [comment]
@byjg No, it doesn't, look [here](https://stackoverflow.com/a/72729190/3261332) please, it seems the `exist()` is causing the issue. Can you tell me if you tested in the script with apache or nginx?

**Comment by byjg** [comment]
Hello @hAbd0u, sorry for these years without answering you. 

I left the project without maintenance for some years, and now I returned with full throttle, not only this, but the whole ecosystem. 

Also, I am bringing new classes, like real `Promises` running asynchronously like Promises in JavaScript.

Apart to the good news, I finally created some constraints to make clear where I can run PHP Threads. 

Unfortunately, it is clear to me that, besides pcntl_fork or parallel pecl component, PHP is not async in essence like JS. So, to have the project running, I had to impose [some limitations](https://github.com/byjg/php-phpthread/blob/master/docs/limitations.md). 

One of them is that Threads doesn't work quite well in the PHP-FPM environment like nginx or apache. It meant to run in CLI. 

I didn't resolve your problem (and probably today isn't matter that); however, I'd like to provide you an answer.

**Comment by hAbd0u** [comment]
Hello @byjg 

Thank you for your clarification and the work that speaks about you. I am totally with you, some things just can't run in web environnement simply because it didn't meant to, so it is best to focus in implementing essential features rather than working around limitations.

Again thank you and I hope you the best.
